src/main/java/ladder/model/Line.java

```diff
@@ -0,0 +1,50 @@
+package ladder.model;
+
+import ladder.model.utils.ConnectLine;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+
+import static java.lang.Boolean.FALSE;
+import static java.lang.Boolean.TRUE;
+
+public class Line {
+    private final List<Boolean> points;
+
+    public Line(List<Boolean> points) {
+        checkValidConnect(points);
+        this.points = points;
+    }
+
+    public static Line createOneLine(int countOfPerson) {
+        List<Boolean> points = new ArrayList<>();
+        int columnNum = countOfPerson - 1;
+
+        while (points.size() < columnNum) {
+            connectLine(points);
+        }
+        if (points.size() > columnNum) {
+            points.remove(points.size() - 1);
+        }
+        return new Line(points);
+    }
+
+    private static void connectLine(List<Boolean> points) {
+        if (ConnectLine.isConnect()) {
+            points.add(TRUE);
+        }
+        points.add(FALSE);
```

Comment:
'가로줄이 이어질 수 없다'는 요구사항에 비해 '라인이 이어져 있다면 라인을 추가'하는 네이밍이 조금 어색한 느낌이에요
이전 값에 따라 다음 가로줄을 결정하면 어떨까요?
```suggestion
        boolean next = ConnectLine.next(before);
        points.add(next);
```